### PR TITLE
Rotate CUP key

### PR DESCRIPTION
**Pull request by mherrmann**

### CODE REVIEW

**Review by simonhong — APPROVED** [review]

File: omaha/net/cup_ecdsa_pubkey.2.h [hunk 1]
```diff
@@ -0,0 +1,31 @@
+// Copyright 2023 Google Inc.
```
Comment (simonhong) [inline omaha/net/cup_ecdsa_pubkey.2.h:1]: Maybe we could use our license for new files? cc @fmarier

Comment (fmarier) [reply omaha/net/cup_ecdsa_pubkey.2.h:1]: It depends where we took that file. If we wrote it ourselves we can choose the license we want, but if we took it from somewhere else, then we need to keep the existing copyright statement.

Comment (mherrmann) [reply omaha/net/cup_ecdsa_pubkey.2.h:1]: The file was auto-generated from a tool in GitHub.com/Google/Omaha and came with this header.

Comment (fmarier) [reply omaha/net/cup_ecdsa_pubkey.2.h:1]: In that case, we should just leave the header as it is.

**Pull request merged**